projects/motor_controller_interface/inc/motor_controller.h

```diff
@@ -0,0 +1,161 @@
+#pragma once
+
+#include <assert.h>
+#include <stdint.h>
+
+// Messages should be sent at least 50ms
+#define MOTOR_CONTROLLER_MESSAGE_INTERVAL_US (40 * 1000)
+
+// Note: All CAN messages that are sent to/from the Motor Controllers have 8
+// bytes of payload.
+
+// 1 cm/s = (1 cm/s) * (1 m/100 cm)
+#define MOTOR_CONTOLLER_VELOCITY_CMPS_TO_MPS(cmps) ((float)(cmps) * ((float)1 / (float)100))
+// 1 m/s = (1 m/s) * (100 cm / 1 m)
+#define MOTOR_CONTROLLER_VELOCITY_MPS_TO_CMPS(mps) ((float)(mps) * ((float)100 / (float)1))
+
+// 1km/h = (1 km/h) * (1 h/60 min) * (1 min/60 s) * (1000 m/1 km) * (100 cm / 1 m)
+#define MOTOR_CONTROLLER_VELOCITY_KMPH_TO_CMPS(kmph)                       \
```

Comment:
You should try to do as much math as possible in integer precision before converting to floats. Our MCU doesn't have an FPU so floating point operations are very slow.